.env.example

```diff
@@ -1,16 +1,24 @@
-# Gemfile
-GEM_PG_VERSION=1.2.2
+# Database
+DB_HOST=postgis
+DB_NAME=gtt
+DB_USER=gtt
+DB_PASS=gtt
+DB_PORT=5432
 
-# docker-compose.yml - postgres
-POSTGRES_USER=gtt
-POSTGRES_PASSWORD=gtt
-POSTGRES_DB=gtt
-
-# config/configuration.yml
-SMTP_ENABLE_STARTTLS_AUTO=true # (or false)
-SMTP_ADDRESS=smtp.gmail.com
-SMTP_PORT=587
-SMTP_DOMAIN=smtp.gmail.com
-SMTP_AUTHENTICATION=plain # (or login)
+# Email
+SMTP_ADDRESS=smtp
+SMTP_DOMAIN=smtp.example.com
+SMTP_PORT=1025
 SMTP_USER_NAME=YOUR_EMAIL_ADDRESS
 SMTP_PASSWORD=YOUR_PASSWORD
+SMTP_ENABLE_STARTTLS_AUTO=true
+
+# Setup
+REDMINE_LANG=en
+REDMINE_EMAIL=gtt@example.com
+
+# System
+RAILS_ENV=production
+RAILS_LOG_TO_STDOUT=1
+RAILS_MEMORY_LIMIT=1024
+PUMA_WORKERS=3
```

Comment:
@dkastl (CC: @smellman, @nobnisai)
This is nice enhancement!

When I set `PUMA_WORKERS=0`, then it seems not to launch multiple workers and killers.
```
docker-gtt-redmine-1  | Puma starting in single mode...
docker-gtt-redmine-1  | * Puma version: 5.6.5 (ruby 3.1.2-p20) ("Birdie's Version")
docker-gtt-redmine-1  | *  Min threads: 0
docker-gtt-redmine-1  | *  Max threads: 5
docker-gtt-redmine-1  | *  Environment: production
docker-gtt-redmine-1  | *          PID: 1
docker-gtt-redmine-1  | * Listening on http://0.0.0.0:3000
docker-gtt-redmine-1  | Use Ctrl-C to stop
```

The following is the default (`PUMA_WORKERS=3`).
```
docker-gtt-redmine-1  | [1] Puma starting in cluster mode...
docker-gtt-redmine-1  | [1] * Puma version: 5.6.5 (ruby 3.1.2-p20) ("Birdie's Version")
docker-gtt-redmine-1  | [1] *  Min threads: 0
docker-gtt-redmine-1  | [1] *  Max threads: 5
docker-gtt-redmine-1  | [1] *  Environment: production
docker-gtt-redmine-1  | [1] *   Master PID: 1
docker-gtt-redmine-1  | [1] *      Workers: 3
docker-gtt-redmine-1  | [1] *     Restarts: (✔) hot (✔) phased
docker-gtt-redmine-1  | [1] * Listening on http://0.0.0.0:3000
docker-gtt-redmine-1  | [1] Use Ctrl-C to stop
docker-gtt-redmine-1  | [1] - Worker 0 (PID: 21) booted in 0.0s, phase: 0
docker-gtt-redmine-1  | [1] - Worker 1 (PID: 23) booted in 0.0s, phase: 0
docker-gtt-redmine-1  | [1] - Worker 2 (PID: 25) booted in 0.0s, phase: 0
```